prov/efa/src/efa_msg.c

```diff
@@ -330,10 +340,7 @@ static ssize_t efa_post_send(struct efa_ep *ep, const struct fi_msg *msg, uint64
 	if (flags & FI_MORE)
 		return 0;
 
-	ret = ibv_post_send(qp->ibv_qp, ep->xmit_more_wr_head.next, &bad_wr);
-
-	free_send_wr_list(ep->xmit_more_wr_head.next);
-	ep->xmit_more_wr_tail = &ep->xmit_more_wr_head;
+	ret = efa_post_flush(ep, &bad_wr);
```

Comment:
I think this can be used in the `out_err` block as well.

Reply:
In `out_err` block, `ibv_post_send()` will only be called when `ep->xmit_more_wr_head.next`  is satisfied, while `free_send_wr_list` and `ep->xmit_more_wr_tail = &ep->xmit_more_wr_head;` does not.